lnwallet/channel.go

```diff
@@ -1340,10 +1340,10 @@ func NewLightningChannel(signer Signer, pCache PreimageCache,
 	// First, initialize the update logs with their current counter values
 	// from the local and remote commitments.
 	localUpdateLog := newUpdateLog(
-		localCommit.LocalLogIndex, localCommit.LocalHtlcIndex,
+		remoteCommit.LocalLogIndex, remoteCommit.LocalHtlcIndex,
```

Comment:
Hmm, I see what you're getting at, but shouldn't this be the _remote pending commitent_? Our pending changes will be committed to that commit diff, not this "locked-in" commitment.

Reply:
Using the remote commit should be fine, as all updates on the pending commit will be appended, not restored, and therefore increment the counters.

Reply:
Yep